sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusConnection.cs

```diff
@@ -381,17 +385,19 @@ internal virtual TransportSender CreateTransportProducer(ServiceBusRetryPolicy r
         /// <param name="ownerLevel">The relative priority to associate with the link; for a non-exclusive link, this value should be <c>null</c>.</param>
         /// <param name="prefetchCount">Controls the number of events received and queued locally without regard to whether an operation was requested.  If <c>null</c> a default will be used.</param>
         /// <param name="sessionId"></param>
+        /// <param name="isSessionReceiver"></param>
         ///
         /// <returns>A <see cref="TransportConsumer" /> configured in the requested manner.</returns>
         ///
         internal virtual TransportConsumer CreateTransportConsumer(
             ServiceBusRetryPolicy retryPolicy,
             long? ownerLevel = default,
             uint? prefetchCount = default,
-            string sessionId = default)
+            string sessionId = default,
+            bool isSessionReceiver = default)
         {
             Argument.AssertNotNull(retryPolicy, nameof(retryPolicy));
-            return InnerClient.CreateConsumer(retryPolicy, ownerLevel, prefetchCount, sessionId);
+            return InnerClient.CreateConsumer(retryPolicy, ownerLevel, prefetchCount, sessionId, isSessionReceiver);
```

Comment:
At some point, we probably want to consider unifying terminology;  if we're calling these "Sender/Receiver" rather than "Producer/Consumer"  we'll probably want to update the member names.